usr/share/rear/backup/NETFS/default/500_make_backup.sh

```diff
@@ -178,9 +224,9 @@ sleep 1
 case "$(basename $BACKUP_PROG)" in
     (tar)
         if (( $backup_prog_rc == 1 )); then
```

Comment:
Not a needed change but some clean up by the way:

Could you replace the several same `$(cat $TMP_DIR/failing_backup_prog)`
with one variable assignment and use that variable value like
<pre>
case "$(basename $BACKUP_PROG)" in
    (tar)
        if (( $backup_prog_rc == 1 )); then
            failed_prog="$( cat $TMP_DIR/failing_backup_prog )"
            LogPrint "WARNING: $failed_prog ended with ...
    ...
</pre>